Resources/Prototypes/Entities/Objects/Tools/toolbox.yml

```diff
@@ -120,6 +120,39 @@
       types:
         Blunt: 20
 
+- type: entity
+  name: singularity bait toolbox
+  parent: BaseItem
+  id: ToolboxSinguloBait
+  description: A toolbox that the singularity really wants to consume.
+  components:
+  - type: Sprite
+    sprite: Objects/Tools/Toolboxes/toolbox_singbait.rsi
+    state: icon
+  - type: Item
+    sprite: Objects/Tools/Toolboxes/toolbox_singbait.rsi
+    size: 9999
```

Comment:
Don't use dummy item sizes.

Reply:
i'll be remaking it into the TG singularity beacon and removing this so probably resolved